wagtail/admin/rich_text/editors/draftail/__init__.py

```diff
@@ -33,16 +33,22 @@ def __init__(self, *args, **kwargs):
 
         super().__init__(*args, **kwargs)
 
+    def translate_value(self, value):
```

Comment:
`format_value` would probably have been a better choice, for consistency with Django's own widget classes, but given that we're subclassing a Django widget I didn't want to risk introducing unexpected changes at this point in the release cycle.